### ISSUE TITLE
System perl used in build process

### ISSUE THREAD

**Issue by eserte** [dod38fr/config-model-lcdproc#1]
Build fails if the current perl != system perl (or the first perl in PATH):

```
Output from './Build':

Executing script/lcdconf2model.pl
Can't locate Config/Model.pm in @INC (you may need to install the Config::Model module) (@INC contains: ... /usr/local/lib/perl5/site_perl/mach/5.20 /usr/local/lib/perl5/site_perl /usr/local/lib/perl5/5.20/mach /usr/local/lib/perl5/5.20 /usr/local/lib/perl5/site_perl/5.20 /usr/local/lib/perl5/site_perl/5.20/mach .) at script/lcdconf2model.pl line 36.
BEGIN failed--compilation aborted at script/lcdconf2model.pl line 36.
gen_LCDd_model failed: 512 at /usr/home/eserte/.cpan/build/2016071318/Config-Model-LcdProc-2.048-5prQYX/_build/lib/Module/Build/Custom.pm line 8.
```

Probably somewhere `$^X` has to be used.

**Comment by dod38fr** [comment]
Ack. I'll fix this.

Thanks for the report

**Comment by dod38fr** [comment]
Fixed in 2.049

Thanks
